presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.234
+=============
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
+* Fix a regression in the new scheduler temporarily by setting the default for the configuration property ``use-legacy-scheduler`` to ``true``.
+* Improve system reliability by preventing additional queries to execute when the total number of tasks exceeds the limit, which can be
+  set by configuration property ``experimental.max-total-running-task-count-to-halt-scheduling``.
+* Improve storage efficiency for Bing tiles by storing Bing tiles as ``BIGINT``. This also reduces bucket skew in certain cases.
+* Improve the scale writer heuristics by considering overall producer buffer utilization. This can be enabled by using the session property
+  ``optimized_scale_writer_producer_buffer`` and the configuration property ``optimized-scale-writer-producer-buffer``.
+* Improve connection pooling to avoid running out of sockets.
+* Add ``KHyperLogLog`` type and related functions.
```

Comment:
We need to add documentation when adding new functions @mcorreaiz  @tdcmeehan  @shixuan-fan 
See issue: https://github.com/prestodb/presto/issues/14306